pandas/tests/plotting/test_converter.py

```diff
@@ -70,15 +70,17 @@ def test_registering_no_warning(self):
         # Set to the "warn" state, in case this isn't the first test run
         register_matplotlib_converters()
         ax.plot(s.index, s.values)
+        plt.close()
 
     def test_pandas_plots_register(self):
-        pytest.importorskip("matplotlib.pyplot")
+        plt = pytest.importorskip("matplotlib.pyplot")
         s = Series(range(12), index=date_range("2017", periods=12))
         # Set to the "warn" state, in case this isn't the first test run
         with tm.assert_produces_warning(None) as w:
             s.plot()
 
         assert len(w) == 0
+        plt.close()
```

Comment:
dont/shouldnt we have a fixture for this?

Reply:
I think there's no need to add a fixture just for `plt = pytest.importorskip("matplotlib.pyplot")` and  `plt.close()`, since the tests between them is quite different. BTW, `plt = pytest.importorskip("matplotlib.pyplot")` showed up only four times in the body of `pandas/tests/plotting/test_converter.py`(three of them will be solved by `plt.close()` and the last one seems to have no influence--Line99 in the current file).

Reply:
makes sense, thanks